src/main/java/jp/co/future/uroborosql/client/command/GenerateCommand.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2017-present, Future Corporation
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package jp.co.future.uroborosql.client.command;
+
+import java.io.PrintWriter;
+import java.sql.SQLException;
+import java.util.Optional;
+import java.util.Properties;
+
+import org.jline.reader.LineReader;
+import org.jline.terminal.Terminal;
+
+import jp.co.future.uroborosql.SqlAgent;
+import jp.co.future.uroborosql.client.completer.SqlKeywordCompleter;
+import jp.co.future.uroborosql.client.completer.SqlKeywordCompleter.SqlKeyword;
+import jp.co.future.uroborosql.client.completer.TableNameCompleter;
+import jp.co.future.uroborosql.config.SqlConfig;
+import jp.co.future.uroborosql.context.SqlContext;
+import jp.co.future.uroborosql.mapping.Table;
+import jp.co.future.uroborosql.mapping.TableMetadata;
+
+/**
+ * Generate sql template Command
+ *
+ * @author H.Sugimoto
+ */
+public class GenerateCommand extends ReplCommand {
+
+	/**
+	 * Constructor
+	 */
+	@SuppressWarnings("unchecked")
+	public GenerateCommand() {
+		super(false, SqlKeywordCompleter.class, TableNameCompleter.class);
+	}
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @see jp.co.future.uroborosql.client.command.ReplCommand#execute(org.jline.reader.LineReader, java.lang.String[], jp.co.future.uroborosql.config.SqlConfig, java.util.Properties)
+	 */
+	@Override
+	public boolean execute(final LineReader reader, final String[] parts, final SqlConfig sqlConfig,
+			final Properties props) {
+		PrintWriter writer = reader.getTerminal().writer();
+		writer.println("GENERATE:");
```

Comment:
@HidekiSugimoto189 
In the case of GenerateCommand, I think that it is unnecessary.

Reply:
OK! fix it.